components/notion/actions/get-block-children/get-block-children.js

```diff
@@ -0,0 +1,46 @@
+const notion = require("../../notion.app");
+
+module.exports = {
+  key: "notion-get-block-children",
+  name: "Get Block Children",
+  description:
+    "Gets a list of child block objects contained in a block given its identifier.",
+  version: "0.0.3",
+  type: "action",
+  props: {
+    notion,
+    blockId: {
```

Comment:
Is an options method here to provide a list possible? https://pipedream.com/docs/components/api/#async-options-example

If not, how does one get a blockId?

Reply:
See my comment about pulling all blocks ids.